keps/sig-apps/2879-ready-pods-job-status/README.md

```diff
@@ -281,7 +284,7 @@ No.
 
   Estimated throughput: at most one API call for each Job Pod reaching Ready
   condition.
-  
+
```

Comment:
Please describe the rate-limitting mechanism (that you mentioned in the comment) here - it's a very important mitigation that has to be mentioned here.

Reply:
Looks good - thanks.